client/src/utils/application/tasksValidation.js

```diff
@@ -1,13 +1,13 @@
 import * as yup from 'yup';
 
 const schema = yup.object().shape({
-  TechnicalTasksLinks: yup
+  technicalTasksLinks: yup
     .string()
     .trim()
-    .required()
+    .required('Enter GitHub link')
```

Comment:
```suggestion
    .required('Enter your GitHub profile link')
```

Reply:
we want Project GitHub in here so I can change it to `Enter your GitHub Project link`

Reply:
yes, exactly